app/models/doctor.rb

```diff
@@ -0,0 +1,6 @@
+class Doctor < ApplicationRecord
+  has_many :appointments
+  has_many :users, :through => :appointments
+  validates :name, format: { with: /\A[a-zA-Z]z/, message: "must not contain special characters" }
```

Comment:
These name regular expressions are not defined properly - I can't enter a standard name.